apps/system/test/unit/lockscreen/lockscreen_passcode_validator_test.js

```diff
@@ -1,55 +0,0 @@
-'use strict';
```

Comment:
I would prefer still have some unit tests for LockScreen to test if it use the `PasscodeHelper` to valid the passcode. You may add it in the `lockscreen_test.js`